jet_tracking/gui/views/jetImageView.py

```diff
@@ -15,8 +15,10 @@ def __init__(self, context, signals):
         self.context = context
         self.camera = ""
         self.mainLayout = QHBoxLayout()
-        self.createImageWidget()
-        self.createEditorWidget()
+        self.imageWidget = None
+        self.editorWidget = None
```

Comment:
To get rid of the annoying message in pycharm that says the value wasn't set in the init hahahaha, sorry. I realize it makes the code needlessly long in a lot of instances...

Reply:
Oh that's a perfectly reasonable reason, it's happened to us all haha. The code length is not a concern here, we just wanted to make sure there's not some hidden interaction we're missing.

I think if you define these in the class definition the static type checker might stop complaining.  (But this change is not so necessary).  Strictly, these shouldn't ever be `None`, since they're necessary to for the class to operate
```python
class JetImageView(QWidget):
    imageWidget: JetImageWidget
    editorWidget: EditorWidget
```